src/main/java/api/messages/GoblintAnalysisResult.java

```diff
@@ -0,0 +1,23 @@
+package api.messages;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The Class GoblintAnalysisResult.
+ * <p>
+ * Corresponding object to the Goblint analyze request response results in JSON.
+ *
+ * @author Karoliine Holter
+ * @since 0.0.3
+ */
+
+public class GoblintAnalysisResult {
+
+    private final List<String> status = new ArrayList<>();

```

Comment:
The status should be a single string, but this appears to be a problem of `@@deriving to_yojson` on Goblint's side.